src/main/resources/templates/db_scripts/views/receiving_history_view.sql

```diff
@@ -3,7 +3,7 @@ OR REPLACE VIEW receiving_history_view AS
 SELECT pieces.id AS id,
        json_build_object(
            'id', pieces.jsonb ->>'id',
-           'caption', pieces.jsonb ->>'caption',
+           'displaySummary', pieces.jsonb ->>'displaySummary',
```

Comment:
need in some way to notify RMB that view is changed to re-create the view, please see examples in Git Annotate to re-create the view for new version

Reply:
Updated `fromModuleVersion` for this view.

Reply:
I don't see changes for that in schema.json